packages/explorer-shared/tsconfig.json

```diff
@@ -16,7 +20,11 @@
     "module": "commonjs",
     "resolveJsonModule": true,
     "esModuleInterop": true,
-    "forceConsistentCasingInFileNames": true
+    "strictPropertyInitialization": false,
+    "forceConsistentCasingInFileNames": true,

```

Comment:
Hmm why was this necessary to define?

Reply:
I think it was there initially anway. It usually added by default from NestJs when creating a new project so I thought this project should have a similar tsconfig as the projects using NestJs.

Reply:
OKay. I'm fine with that